synthetics_incrementals.tf

```diff
@@ -2,15 +2,12 @@ resource "datadog_synthetics_test" "incrementalsjenkinsio" {
   type = "api"
   request_definition {
     method = "GET"
-    url    = "https://incrementals.jenkins.io/"
+    url    = "https://incrementals.jenkins.io/readiness"
```

Comment:
```suggestion
    # https://github.com/jenkins-infra/incrementals-publisher/blob/a754f43d44be5f6b09e2ac3f9e5600e04175936b/index.js#L62
    url    = "https://incrementals.jenkins.io/readiness"
```

Reply:
@dduportal Other monitors don't have this kind of comment, not sure why adding this comment here?

Reply:
The idea is to start "linking" between repositories. This PR is the opportunity to start this.